package.json

```diff
@@ -1,21 +1,122 @@
+    "not IE 11"
+  ],
+  "jest": {
+    "moduleNameMapper": {
+      "^@/(.*)$": "<rootDir>/$1"
     },
-    "devDependencies": {
-        "laravel-mix": "^6.0.25",
-        "laravel-mix-polyfill": "^3.0.1",
-        "postcss": "^8.1.14",
-        "resolve-url-loader": "^4.0.0",
-        "sass": "^1.35.2",
-        "sass-loader": "^12.1.0"
-    }
+    "modulePathIgnorePatterns": [
+      ".buddy-tests"
+    ]
+  },
+  "dependencies": {
+    "@alleyinteractive/block-editor-tools": "^0.0.1",
+    "@wordpress/a11y": "3.6.1",
+    "@wordpress/annotations": "2.6.2",
+    "@wordpress/api-fetch": "6.3.1",
+    "@wordpress/autop": "3.6.1",
+    "@wordpress/blob": "3.6.1",
+    "@wordpress/block-directory": "3.4.11",
+    "@wordpress/block-editor": "8.5.8",
+    "@wordpress/block-library": "7.3.11",
+    "@wordpress/block-serialization-default-parser": "4.6.1",
+    "@wordpress/blocks": "11.5.3",
+    "@wordpress/components": "19.8.4",
+    "@wordpress/compose": "5.4.1",
+    "@wordpress/core-data": "4.4.5",
+    "@wordpress/customize-widgets": "3.3.11",
+    "@wordpress/data": "6.6.1",
+    "@wordpress/data-controls": "2.6.1",
+    "@wordpress/date": "4.6.1",
+    "@wordpress/deprecated": "3.6.1",
+    "@wordpress/dom": "3.6.1",
+    "@wordpress/dom-ready": "3.6.1",
+    "@wordpress/edit-post": "6.3.11",
+    "@wordpress/edit-site": "4.3.11",
+    "@wordpress/edit-widgets": "4.3.11",
+    "@wordpress/element": "4.4.1",
+    "@wordpress/escape-html": "2.6.1",
+    "@wordpress/format-library": "3.4.8",
+    "@wordpress/hooks": "3.6.1",
+    "@wordpress/html-entities": "3.6.1",
+    "@wordpress/i18n": "4.6.1",
+    "@wordpress/icons": "8.2.2",
+    "@wordpress/interface": "4.5.5",
+    "@wordpress/is-shallow-equal": "4.6.1",
+    "@wordpress/keyboard-shortcuts": "3.4.1",
+    "@wordpress/keycodes": "3.6.1",
+    "@wordpress/list-reusable-blocks": "3.4.4",
+    "@wordpress/media-utils": "3.4.1",
+    "@wordpress/notices": "3.6.1",
+    "@wordpress/nux": "5.4.4",
+    "@wordpress/plugins": "4.4.2",
+    "@wordpress/primitives": "3.4.1",
+    "@wordpress/priority-queue": "2.6.1",
+    "@wordpress/redux-routine": "4.6.1",
+    "@wordpress/reusable-blocks": "3.4.8",
+    "@wordpress/rich-text": "5.4.2",
+    "@wordpress/server-side-render": "3.4.5",
+    "@wordpress/shortcode": "3.6.1",
+    "@wordpress/token-list": "2.6.1",
+    "@wordpress/url": "3.7.1",
+    "@wordpress/viewport": "4.4.1",
+    "@wordpress/warning": "2.6.1",
+    "@wordpress/widgets": "2.4.8",
+    "@wordpress/wordcount": "3.6.1",
+    "check-node-version": "^4.2.1",
+    "classnames": "^2.3.1",
+    "dompurify": "^2.3.4",
+    "lodash": "4.17.21",
+    "moment": "^2.29.4",
+    "papaparse": "^5.3.1",
+    "prop-types": "^15.8.0",
+    "react": "17.0.2",
+    "react-dom": "17.0.2",
+    "styled-components": "^5.3.1",
+    "uuid": "^8.3.2"
+  },
+  "devDependencies": {
+    "@types/jest": "^29.0.3",
+    "@babel/preset-react": "^7.18.6",
+    "@tinypixelco/laravel-mix-wp-blocks": "^1.2.0",
```

Comment:
This should handle all of the @wordpress packages for us, no?

Reply:
What I struggle with is IDE completion. Without these packages, I can't get it unless I use a `@types/` or something. Will dig in.